docs/getting-started.md

```diff
@@ -26,6 +26,24 @@ All grove operator Make targets are located in [Operator Makefile][./operator/Ma
   export KUBECONFIG=./hack/kind/kubeconfig
   ```
 
+### Kubernetes cluster set-up
+
+If you want to use your own Kubernetes cluster instead of the KIND cluster, follow these steps:
+
+- **Set the KUBECONFIG environment variable** to point to your Kubernetes cluster configuration:
+
+   ```bash
+   # Set KUBECONFIG to use your Kubernetes cluster kubeconfig
+   export KUBECONFIG=/path/to/your/kubernetes/kubeconfig
+   ```
+
+- **Set the DEFAULT_REGISTRY environment variable** to specify your Docker image registry:
```

Comment:
Ideally we should call this `CONTAINER_REGISTRY` and if not specified then we should fallback to the `DEFAULT_CONTAINER_REGISTRY`. Using `DEFAULT` gives an impression that there is a provision to overwrite that default as well which is not the case.

Reply:
ok, CONTAINER_REGISTRY could be better understanding.